composer.json

```diff
@@ -11,7 +11,8 @@
         }
     ],
     "require": {
-        "symfony/framework-bundle": "2.*",
+        "php": "^5.5|~7.0",
+        "symfony/framework-bundle": "~2.8|^3.0",
```

Comment:
It isn't really clean to use this constraint as symfony 2.3 and 2.7 are still supported...